child_compassion/models/project_reservation.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class ProjectReservation(models.Model):
+
+    _name = 'compassion.reservation'
+    _description = 'Project Reservation'
+
+    beneficiary_global_id = None
+    channel_name = fields.Char(string='Channel Name', required=True, size=64)
+    global_partner_id = 'CH'
+    icp_id = fields.Many2one(
+        'compassion.project', string='Project Compassion', required=True
```

Comment:
You don't need to specify the name of the parameter when you follow the order of the optional parameters (see the documentation for fields). So here you can omit `string=`. You can also just name it `Project`